server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -48,16 +48,19 @@
         long total = -1;
         long free = -1;
         long available = -1;
+        @Nullable
+        Boolean isWritable;
```

Comment:
Sure but when we actually have a read-only node removed, it would still stay around unless an operator intervenes by either fixing some of the issues or replacing it. I feel `/_nodes/_local/stats` might still serve a good purpose and would let the system know it needs an attention.

Reply:
I think this is already handled by the cluster health API -- the faulty node will report `red` health when it is removed from the cluster which is a much clearer indication that action is needed, and we can record helpful details in the logs since we always check the logs in this kind of situation.

Reply:
While I understand we don't want to expose the stats, is the interface `FsService#stats().getTotal().isWritable() ` acceptable at multiple places that I have used to reject requests @ `PrevoteCollector/JoinHelper/FollowersChecker` or should FsService expose another interface without touching `FsInfo` at all

Reply:
I'll change this to FsService.FsHealthService#isWritable()

Reply:
Yes, I don't think `FsService` should be involved here. `FsHealthService` makes sense on its own.